superset-frontend/src/views/CRUD/welcome/ChartTable.tsx

```diff
@@ -59,6 +61,9 @@ function ChartTable({
   showThumbnails,
 }: ChartTableProps) {
   const history = useHistory();
+  const filterStore = getFromLocalStorage(HOMEPAGE_CHART_FILTER, null);
+  const defaultFilter = filterStore || TableTabTypes.MINE;
```

Comment:
suggestion: `initialFilter` would be a more appropriate name. "default" typically indicates that it will be the value used when nothing is selected, but in this UI the user cannot select nothing.